src/components/Form.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <form :action="send_url" :method="send_method" @submit.prevent="send">
+        <slot name="form-errors">
+            {{ form_errors.join(', ') }}
+        </slot>
+        <slot name="autogenerated-items">
+            <section v-for="(field, key) in form_fields" :key="key">
+                <FormItem :ref="field.name" v-model="form_fields[key].value" :field="field"/>
+            </section>
```

Comment:
It's quite unusual to use `section` tag for individual form elements. It can also create some clashes in CSS, so I'd advise against it.

Reply:
Fixed